glide.yaml

```diff
@@ -39,6 +39,8 @@ import:
 - package: golang.org/x/crypto
   subpackages:
   - ssh
+- package: github.com/alexcesaro/statsd
+  version: ^2.0.0
```

Comment:
Legal rules are different for public facing projects versus internal Microsoft projects. This version was not registered with Microsoft CELA yet. I went ahead and did so: https://ossmsft.visualstudio.com/_oss?requestId=41301&_a=progress

In the future you'll need to register any new OSS component or any OSS version change here: https://ossmsft.visualstudio.com/_oss (requires MSFT login).